modules/microshift-exposed-audit-ports.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-exposed-audit-ports_{context}"]
+= Auditing exposed network ports 
+
+On {microshift-short}, the host port can be opened by a workload in the following cases. You can check logs to see view the network services.
+//Might need a better description here
```

Comment:
Also not sure if this description is enough

Reply:
It looks good to me.